script.screensaver.themealdb/resources/language/English/strings.po

```diff
@@ -0,0 +1,141 @@
+# Kodi Media Center language file
```

Comment:
please use the new layout for the language folder: `/resources/language/resource.language.en_gb/strings.po`